pkg/arvo/sys/vane/ames.hoon

```diff
@@ -2391,6 +2392,16 @@
     =.  message-pump  (give i.packet-pump-gifts)
     ::
     $(packet-pump-gifts t.packet-pump-gifts)
+  ::  +assert: sanity checks to isolate error cases
```

Comment:
Actually, I wonder if we'd be better off just printing here to make sure ships don't get stuck from crashing on this -- especially since you added the recovery logic.